Makefile

```diff
@@ -38,6 +38,10 @@ generate:
 format:
 	gofumpt -l -w .
 
+.PHONY: lint
+lint:
+	golangci-lint run -c ./.golangci.yml  ./...
```

Comment:
Both the -c option and the `./...` argument are redundant, so I'd prefer a plain `golangci-lint run`.

Reply:
Thanks. I took the liberty of squashing this into the first commit, it doesn't make sense to me to have these as two commits in the commit history. (A fixup commit would have been a good tool in this case, see https://github.com/jesseduffield/lazygit/blob/master/docs/Fixup_Commits.md for some background on that. For next time 😄)

Reply:
Thanks for the comment. 😄
I know fixup commit but I don't have the habit of using them. I will try it next time! 👍 

By the way, if you use make target frequently, my tool will help you. If you're interested please take a look!

https://github.com/kyu08/fzf-make